### ISSUE TITLE
Installation breaks project

### ISSUE THREAD

**Issue by qwetboy10** [leisim/logger_flutter#1]
Adding the dependency to my project causes flutter run to spit out this error.
```
FAILURE: Build failed with an exception.

* Where:
Script '/home/tristan/Files/ProgramFiles/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 221

* What went wrong:
A problem occurred configuring project ':logger_flutter'.
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':logger_flutter' of type org.gradle.api.Project.
   > Could not find method debugProvided() for arguments [file collection] on object of type
   org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run
with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
  Command: /home/tristan/Projects/CyWoodsAppFlutter/android/gradlew app:properties


Please review your Gradle project setup in the android/ folder.
```

**Comment by ssilverstein** [comment]
I get the same error

**Comment by joeblew99** [comment]
check the logs to see whats wrong then :) Sorry could not help myself.

**Comment by Daan-** [comment]
It looks like this project is missing all required Android files. The pubspec declares:
    androidPackage: com.github.leisim.logger_flutter
    pluginClass: LoggerFlutterPlugin

However none of these are present. I guess they should be added to Git?

**Comment by certhon** [comment]
I get the same error

**Comment by softcoder** [comment]
I get the same error, ONLY when i add this dep to my pubspec.yaml, last 2 lines:

  logger: ^0.7.0+2
  logger_flutter: ^0.7.0


A problem occurred configuring project ':logger_flutter'.
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':logger_flutter' of type org.gradle.api.Project.
   > Could not find method debugProvided() for arguments [file collection] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.


Note: changing logger to 0.7.0 does not fix this, removing logger_flutter makes the project compile fine.

**Comment by aahamlin** [comment]
I am facing same issue only on iOS, logger_flutter plugin is added to .flutter-plugins file but cannot be installed because the project does not include ios podspec file. From the git repo, it looks like it also does not include any Android files either (as a previous comment above reports). Is there a command or configuration in flutter to install a package and bypass the attempts to install platform specific code when none exists?

**Comment by simc** [comment]
Thanks for reporting and sorry for the late reply. I fixed this in version 0.7.1.
